django/utils/http.py

```diff
@@ -293,12 +295,21 @@ def is_safe_url(url, host=None):
             url = force_text(url)
         except UnicodeDecodeError:
             return False
+    if allowed_hosts is None:
+        allowed_hosts = set()
+    if host:
+        warnings.warn(
```

Comment:
`hosts` is along with the existing argument and defines it's plural form 😉 

I agree that `allowed_hosts` on its own is a better name, but we would have to rename `host` to `allowed_host` to keep a consistent naming.

My 2¢ 😃